### PR TITLE
Feat/test workflow

### PR DESCRIPTION
**Pull request by ngunner15**
- Add workflow for all 3 environment(Test, Dev, Prod)
- The trigger comes from - [vault-sync-config](https://github.com/bcgov-nr/vault-sync-config/blob/main/.github/workflows/vault-sync-config.yml) repo
- Use [Trigger External Workflow](https://github.com/marketplace/actions/trigger-workflow-and-wait) template to achieve Cross Repository Triggering

### CODE REVIEW

File: .github/workflows/vault-sync-app-body.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: runner-config
+on:
```
Comment (mbystedt) [inline .github/workflows/vault-sync-app-body.yml:2]: Should be able dispatch manually as well

File: .github/workflows/vault-sync-app-body.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  workflow_call:
+    inputs:
+      name:
```
Comment (mbystedt) [inline .github/workflows/vault-sync-app-body.yml:5]: The input is the environment (not name)

File: .github/workflows/vault-sync-app-body.yml (hunk 1 above)
Comment (ngunner15) [reply .github/workflows/vault-sync-app-body.yml:2]: So add workflow_dispatch?

Comment (mbystedt) [reply .github/workflows/vault-sync-app-body.yml:2]: yes

Comment (mbystedt) [reply .github/workflows/vault-sync-app-body.yml:2]: Will the workflow_dispatch need inputs as well?

**Review by mbystedt — APPROVED** [review]

**Pull request merged**